src/services/user-settings/edit-excluded-categories.ts

```diff
@@ -13,15 +13,7 @@ export const editExcludedCategories = withTransaction(
   }): Promise<number[]> => {
     const [existingSettings] = await UserSettings.findOrCreate({
       where: { userId },
-      defaults: {
-        settings: { 
-          stats: {
-            expenses: {
-              excludedCategories: []
-            }
-          } 
-        },
-      },
+      defaults: { ZodSettingsSchema },
```

Comment:
`ZodSettingsSchema` returns Zod instance, but not object it describes. This simply won't work.